en/appendices/4-5-migration-guide.rst

```diff
@@ -80,6 +80,9 @@ Log
 ORM
 ---
 
+- In CakePHP 5 ``EntityTrait::has()`` will return true when an attribute exists
+  and is set to null. To keep the old behavior it is recommended to
+  change ``EntityTrait::has()`` to ``EntityTrait::hasValue()``.
```

Comment:
I don't think this is true. hasValue() has completely different behavior than has().

Reply:
Sure it ain't a 1:1 replacement. What would be the recommended replacement to not get warnings in the template for already baked templates? Keeping it as it is will result in warnings being printed.

Reply:
We just can't say "To keep the old behavior".  Need to explain there's an alternative that returns false on null but has other behavior too.

Reply:
Thats true. I tried to reword it.